runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -135,8 +136,13 @@ public Token appendCheckpoint() {
         // Queries the sequencer for the stream tail. We then read the stream tail to
```

Comment:
Can you change the comment accordingly? Because now we also need global tail. And also it would be better to explain this query api guarantees no updates between stream tail and global tail.

Reply:
Done.